terminusdb_client/woqlclient/woqlClient.py

```diff
@@ -164,29 +444,32 @@ def create_database(
         prefixes=None,
         include_schema=True,
     ):
-        """
-        Create a Terminus Database by posting
-        a terminus:Database document to the Terminus Server
+        """Create a TerminusDB database by posting
+        a terminus:Database document to the Terminus Server.
 
         Parameters
         ----------
         dbid : str
-            ID of the specific database to create
-        label: textual name of db
-        description: text description of db
-        prefixes: prefixes
-        key : str, optional
-            you can omit the key if you have set it before
-        kwargs
-            Optional arguments that ``createDatabase`` takes
+            Unique identifier of the database.
+        accountid : str
+            Account identifier.
+        label : str, optional
+            Database name.
+        description : str, optional
+            Database description.
+        prefixes
+            TODO
+        include_schema : bool
+            TODO
```

Comment:
I am not sure how do describe these parameters. If `prefixes` is set then it goes unused. Any suggestions?

Reply:
prefixes (dict - optional) dict containing:
            doc: (string) the IRI to use when doc: prefixes are expanded (defaults to terminusdb:///data)
            scm: (string) the IRI to use when scm: prefixes are expanded (defaults to terminusdb:///schema)

Reply:
Also, there's a small bug, it should be just "schema" not "include_schema" 

        schema (boolean - optional) if set to true, a main schema graph will be created, if not set, only a main instance graph will be created

Reply:
Thanks!